soda_machine.rb

```diff
@@ -7,12 +7,26 @@ def initialize(args = {})
   end
 
   def current_inventory_count
+    @sodas.length
   end
 
   def find_soda(soda_brand)
+    soda_check = @sodas.each do |soda|
```

Comment:
this assignment makes no sense.  why are you saving `soda_check` here?  you can just iterate over the collection and return `nil` at the end of the method.  this would satisfy the tests here.